src/script.js

```diff
@@ -0,0 +1,39 @@
+const bscript = require('bscript-parser')
+
+class Script {
+  get length () {
+    return this.serialized.length
+  }
+
+  constructor (serializedScript) {
```

Comment:
What is serializedScript here?

Reply:
A buffer of all of the bytes in the script